data/card_catalogue/weapons.json

```diff
@@ -5,38 +5,37 @@
     "tier": "green",
     "details": {
       "rules": {
-        "passive": "Melee attacks knock the target back 3 squares",
-        "active": "Throw a heavy sledgehammer at a visible target"
```

Comment:
Removed the active so it's better suited for the list of:
- #106 

Begs the question - could you throw it anyway? It has only got a passive and in this case it'd make sense that even a non-cardmaster could pick it up and have the passive effect work for them. Maybe best to leave that as an emergent mechanic so we don't have to go back to formally differentiating *Item*- and *Action*-type cards.

Reply:
Hm, I don't think we can really have a card without an active since it'll break the improvise set. I think if we don't have an active in mind for this we should nix it, or work out this new design pattern in a separate issue.

What about `Knock a character back 5 squares`?

Reply:
I'll cut the card for now since I think this pertains to 

- #112